codegenerator/cli/templates/static/codegen/src/Address.res

```diff
@@ -1 +1,22 @@
-type t = Ethers.ethAddress
\ No newline at end of file
+@genType.import(("./OpaqueTypes.ts", "Address"))
+type t
+
+let schema =
+  S.string->S.setName("Address")->(Utils.magic: S.t<string> => S.t<t>)
+
+external toString: t => string = "%identity"
+
+external unsafeFromString: string => t = "%identity"
+
+module Evm = {
+  @module("ethers") @scope("ethers")
+  external fromStringOrThrow: string => t = "getAddress"
+
+  /**
+  Same binding as getAddress from string 
+  but used when we receive and address that's not necessarily checksummed
+  */
+  @module("ethers")
+  @scope("ethers")
+  external checksum: t => t = "getAddress"
```

Comment:
This should no longer be necessary. If it's used anywhere we should remove it. I think it's a hack because hypersync client was not checksumming addresses. Otherwise it can add quite a lot of overhead.

Reply:
I'll do in another PR